Makefile

```diff
@@ -66,8 +66,6 @@ start-devserver: ## Starts the development server
 	@printf 'Please wait while the development server starts...\n\n'
 	@while [[ $$(curl -s -o .dev/status_code.txt -w '%{http_code}' http://localhost:8181) != "200" ]] || [[ $$(curl -s -o .dev/status_code.txt -w '%{http_code}' http://localhost:8181/community-library) != "200" ]]; do \
 		printf "▓"; \
-		if [[ "$(prod_env)" = 'true' ]] && [[ -n $$(docker ps -q -f status=exited -f name=webpack-compiler) ]]; then \
-			${SHELL_PREFIX} dev-server python -m scripts.generate_build_directory; \
 		fi; \
```

Comment:
You have to drop the fi too, it goes with the if.

Reply:
Done